CONTRIBUTING.md

```diff
@@ -6,24 +6,26 @@
 
     For language-specific issues with the "Default Language Packages" (which handle syntax highlighting and other language-related things), refer to the [Default Language Packages repository](https://github.com/sublimehq/Packages).
 
-    For issues related to other packages, head over to the respective repository or contact its author. Start with a [clean profile](http://www.sublimetext.com/docs/3/revert.html) if you need to verify.
+    For issues related to other packages, head over to the respective repository or contact its author.
+    Start with a [clean profile](http://www.sublimetext.com/docs/revert.html) if you need to verify.
 
-*   Search for the issue [here](https://github.com/sublimehq/sublime_text/search?q=&type=Issues) to check if it was already reported. You may use labels for filtering the [issue list](https://github.com/sublimehq/sublime_text/issues) by clicking any of these related to the problem you want to report or request.
+*   Search for the issue [here](https://github.com/sublimehq/sublime_text/search?q=&type=Issues) to check if it was already reported.
```

Comment:
Undecided if linking to this search page is a good idea. The search input is really far await from the main content (the issues) and I personally always search for issues via the issue list. It also allows to view the most recently created (and thus probably also more relevant after new releases) issues first.

What do others think?

Reply:
For clarification, the search input on the top right (cut off in your screenshot) should be used here and submitting it will update the search results, limiting them to issues as they already are.

Reply:
Oh, true, I misread the diff.

I think going forward we should rephrase this to link to the issue list in this line and in the line below simply refer to it instead of linking it again.